exercises.md

```diff
@@ -50,16 +50,22 @@ makes two input-output assertions based on some extremal points for which it is
 
 5. (Bonus) Check the coverage of the whole test suite:
 
-```
+```bash
 pytest tests/ --cov src
 ```
 
 You can get a report on which lines are not visited by any tests with:
 
-```
+```bash
 pytest tests/ --cov src --cov-report term-missing
 ```
 
+Or if you prefer more visual report, you can try to generate HTML report
+```bash
+pytest tests/ --cov src --cov-report html
+```
+this will generate `htmlcov` folder where you can see the report in your browser by opening `htmlcov/index.html`
+
```

Comment:
Though some people may like to  inspect coverage reports in html, so I added a short section how to generate it.